files/fluentd/run.sh

```diff
@@ -68,8 +67,31 @@ fi
 # Determine allowed total given the number of outputs we have.
 ALLOWED_DF_LIMIT=$(expr $DF_LIMIT \* $ALLOWED_PERCENT_OF_DISK / 100) || :
 
+TOTAL_LIMIT_SIZE_CONF=$(grep "total_limit_size" $FLUENT_CONF |  head -1 | awk '{print $2}' )
```

Comment:
Maybe we should name this either `TOTAL_LIMIT_SIZE_REQUEST_PER_BUFFER` or `TOTAL_LIMIT_SIZE_SPEC_PER_BUFFER`